### PR TITLE
Multi processor

### PR DESCRIPTION
**Pull request by will0x0909**

### CODE REVIEW

File: indexer/utils/provider.py [hunk 1]
```diff
@@ -72,7 +72,11 @@ def make_request(self, method=None, params=None):
         else:
             request_data = params
         raw_response = make_post_request(self.endpoint_uri, request_data, **self.get_request_kwargs())
-        response = self.decode_rpc_response(raw_response)
+        try:
+            response = self.decode_rpc_response(raw_response)
+        except JSONDecodeError:
+            self.logger.error("aaaaa JSON decode error, params: %s, raw_response: %s", params, raw_response)
```
Comment (shanshuo0918) [inline indexer/utils/provider.py:78]: remove aaaa

File: indexer/controller/stream_controller.py [hunk 2]
```diff
@@ -24,18 +29,20 @@ def __init__(
         sync_recorder: BaseRecorder,
         job_scheduler: JobScheduler,
         limit_reader: LimitReader,
-        max_retries=5,
+        max_retries=1,
         retry_from_record=False,
         delay=0,
+        _manager=None,
```
Comment (shanshuo0918) [inline indexer/controller/stream_controller.py:35]: Not in use?

File: indexer/controller/stream_controller.py [hunk 3]
```diff
@@ -2,19 +2,24 @@
...
-from indexer.utils.exception_recorder import ExceptionRecorder
 from indexer.utils.limit_reader import LimitReader
 from indexer.utils.sync_recorder import BaseRecorder
 
-exception_recorder = ExceptionRecorder()
+# exception_recorder = ExceptionRecorder()
 
 logger = logging.getLogger(__name__)
 
+M_JOBS: int = int(os.environ.get("M_JOBS", 4))
```
Comment (shanshuo0918) [inline indexer/controller/stream_controller.py:19]: default  = 1

File: indexer/controller/stream_controller.py [hunk 4]
```diff
@@ -2,19 +2,24 @@
...
+M_TIMEOUT: int = int(os.environ.get("M_TIMEOUT", 100))
```
Comment (shanshuo0918) [inline indexer/controller/stream_controller.py:20]: default to M_SIZE * 300

File: indexer/controller/stream_controller.py [hunk 5]
```diff
@@ -2,19 +2,24 @@
...
+M_SIZE: int = int(os.environ.get("M_SIZE", 100))
```
Comment (shanshuo0918) [inline indexer/controller/stream_controller.py:21]: default to B / M_JOBS

Comment (shanshuo0918) [reply indexer/controller/stream_controller.py:21]: ADD to cli

**Comment by shanshuo0918** [comment]
tgdm

**Comment by will0x0909** [comment]
> tgdm

added

**Review by shanshuo0918 — APPROVED** [review]

**Pull request merged**